<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>728</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(18, 130, 119)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>236</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="input_data">
     <item>
      <widget class="QLineEdit" name="height">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weight">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="age">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:  rgb(185, 196, 201)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="label_data">
     <item>
      <widget class="QLabel" name="height_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:  rgb(185, 196, 201)</string>
       </property>
       <property name="text">
        <string>рост, см</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="weight_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:  rgb(185, 196, 201)</string>
       </property>
       <property name="text">
        <string>вес, кг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="age_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:  rgb(185, 196, 201)</string>
       </property>
       <property name="text">
        <string>возраст</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>белки, г</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="protein">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:  rgb(185, 196, 201)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fats">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:  rgb(185, 196, 201)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="carbohydrates">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:  rgb(185, 196, 201)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>жиры, г</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>углеводы, г</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>энергия, ккал</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="calories">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 0, 0)</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="eat_today">
     <attribute name="title">
      <string>Съедено за день</string>
     </attribute>
     <widget class="QLineEdit" name="value_calories_1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:  rgb(185, 196, 201)</string>
      </property>
      <property name="text">
       <string>0 ккал</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="need_to_eat_today">
     <attribute name="title">
      <string>Осталось съесть за день</string>
     </attribute>
     <widget class="QLineEdit" name="value_calories_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:  rgb(185, 196, 201)</string>
      </property>
      <property name="text">
       <string>0 ккал</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="diet">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>Рацион</string>
    </property>
   </widget>
   <widget class="QPushButton" name="water">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>Вода</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statistic">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>590</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index_weight_body">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>ИМТ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poor Richard</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(185, 196, 201)</string>
    </property>
    <property name="text">
     <string>MyDiet</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
